draft-detecting-unwanted-location-trackers.md

```diff
@@ -348,7 +348,7 @@ The Protocol Implementation Version operand contains a value indicating an imple
 
 
 The Major.Minor.Revision value associated with this document is 1.0.0.
-The equivalent 4-byte value is 0x00000001.
+The equivalent 4-byte value is 0x00010000.
```

Comment:
Is this the right value?
byte 0 (revision number) = 00
byte 1 (minor) = 00
bytes 2-3 (major) = 0001
So the version in big endian is 0x00000001 (defined as Uint32)
Shouldn't the little endian value be 0x01000000

Reply:
Also, should we include the example, or a reference to the same example, in the FW version section below?